File: src/androidTest/java/com/justforfun/simplexml/model/java_models/TestResourcesEntryWithNumbers.java
package com.justforfun.simplexml.model.java_models;

import com.justforfun.simplexml.annotation.XmlName;

/**
 * Created by Vladimir on 6/16/17.
 */
@XmlName(name = "resources")
public class TestResourcesEntryWithNumbers {
    @XmlName(name = "app_name") public String app_name;
    @XmlName(name = "app_version") public int app_version;
    @XmlName(name = "build_info") public TestBuildInfoWithNumbers buildInfo;

    @Override
    public String toString() {
        return "TestResourcesEntryWithNumbers{" +
                "app_name='" + app_name + '\'' +
                ", app_version=" + app_version +
                ", buildInfo=" + buildInfo +
                '}';
    }
}

File: src/androidTest/java/com/justforfun/simplexml/model/kt_models/TestResourcesEntryWithNumbers.kt
package com.justforfun.simplexml.model.kt_models

import com.justforfun.simplexml.annotation.XmlName

/**
 * Created by Vladimir on 6/16/17.
 */
@XmlName("resources")
class TestResourcesEntryWithNumbers {
    @XmlName(name = "app_name") lateinit var app_name: String
    @XmlName(name = "app_version") var app_version: Int? = 0
    @XmlName("build_info") lateinit var buildInfo: TestBuildInfoWithNumbers
}

@XmlName("build_info")
class TestBuildInfoWithNumbers {
    @XmlName("app_code_version") var app_code_version: Int? = 0
    @XmlName("build_version") var build_version: Int? = 0
}